osu.Game/Skinning/Editor/SkinEditor.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Cursor;
+
+namespace osu.Game.Skinning.Editor
+{
+    public class SkinEditor : FocusedOverlayContainer
```

Comment:
This can happen as a follow-up PR if that's decided to be the correct direction. I don't want to dwell on issues like this in this PR, as that's not intended to be the focus here.